storage/testonly/admin_storage_tester.go

```diff
@@ -202,10 +205,7 @@ func (tester *AdminStorageTester) TestUpdateTree(t *testing.T) {
 	ctx := context.Background()
 	s := tester.NewAdminStorage()
 
-	unrelatedTree, err := createTree(ctx, s, MapTree)
-	if err != nil {
-		t.Fatalf("createTree() = (_, %v), want = (_, nil)", err)
-	}
+	unrelatedTree := makeTreeOrFail(ctx, t.Fatalf, s, spec{Tree: MapTree})
```

Comment:
Might read better with the `failFn` as the final argument.

Reply:
Done.